diff --git a/desktop/armagetronad.appdata.xml.in b/desktop/armagetronad.appdata.xml.in
index 0fe41b945..edd333e69 100644
--- a/desktop/armagetronad.appdata.xml.in
+++ b/desktop/armagetronad.appdata.xml.in
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-  <id>@progid@.desktop</id>
+  <id>@progid@</id>
   
   <name>@progtitle@</name>
   <summary>3D Lightcycle Game</summary>
