diff --git a/desktop/armagetronad.appdata.xml.in b/desktop/armagetronad.appdata.xml.in
index fb8e34ff..dcbc0c49 100644
--- a/desktop/armagetronad.appdata.xml.in
+++ b/desktop/armagetronad.appdata.xml.in
@@ -24,16 +24,24 @@
   <screenshots>
     <screenshot type="default">
-      <image>http://www.armagetronad.org/screenshots/ss_fort_1.png</image>
+      <image>https://www.armagetronad.org/screenshots/ss_fort_1.png</image>
     </screenshot>
     <screenshot>
-      <image>http://www.armagetronad.org/screenshots/ss_sumo_1.png</image>
+      <image>https://www.armagetronad.org/screenshots/ss_sumo_1.png</image>
     </screenshot>
     <screenshot>
-      <image>http://www.armagetronad.org/screenshots/screenshot_23.png</image>
+      <image>https://www.armagetronad.org/screenshots/screenshot_23.png</image>
     </screenshot>
   </screenshots>
 
-  <url type="homepage">http://www.armagetronad.org</url>
+  <url type="homepage">https://www.armagetronad.org</url>
   <url type="bugtracker">https://gitlab.com/armagetronad/armagetronad/-/issues</url>
 
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="0.2.9.1.0" date="2020-11-29"/>
+  </releases>
+
 </component>
