diff --git a/desktop/armagetronad.appdata.xml.in b/desktop/armagetronad.appdata.xml.in
index fb8e34ff..dcbc0c49 100644
--- a/desktop/armagetronad.appdata.xml.in
+++ b/desktop/armagetronad.appdata.xml.in
@@ -36,4 +36,12 @@
   <url type="homepage">http://www.armagetronad.org</url>
   <url type="bugtracker">https://gitlab.com/armagetronad/armagetronad/-/issues</url>
 
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="0.2.9.1.0" date="2020-11-29"/>
+  </releases>
+
 </component>
